File: app/src/main/java/com/intelegain/agora/model/HolidayData.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by suraj.m on 24/7/17.
 */

public class HolidayData {

    @SerializedName("ImageId")
    @Expose
    private int ImageId;
    @SerializedName("HolidayDay")
    @Expose
    private String HolidayDay;
    @SerializedName("HolidayMonth")
    @Expose
    private String HolidayMonth;
    @SerializedName("HolidayName")
    @Expose
    private String HolidayName;
    @SerializedName("HolidayImageUrl")
    @Expose
    private String HolidayImageUrl;
    @SerializedName("HolidayBgColor")
    @Expose
    private String HolidayBgColor;

    public HolidayData(String holidayDay, String holidayMonth, String holidayName,
                       String holidayImageUrl, int imageId, String holidayBgColor) {
        HolidayDay = holidayDay;
        HolidayMonth = holidayMonth;
        HolidayName = holidayName;
        HolidayImageUrl = holidayImageUrl;
        ImageId = imageId;
        HolidayBgColor = holidayBgColor;
    }

    public String getHolidayDay() {
        return HolidayDay;
    }

    public String getHolidayMonth() {
        return HolidayMonth;
    }

    public String getHolidayName() {
        return HolidayName;
    }

    public String getHolidayImageUrl() {
        return HolidayImageUrl;
    }

    public int getImageId() {
        return ImageId;
    }

    public String getHolidayBgColor() {
        return HolidayBgColor;
    }

}

File: app/src/main/java/com/intelegain/agora/model/HolidayData.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by suraj.m on 24/7/17.
 */
class HolidayData(@field:Expose @field:SerializedName("HolidayDay") val holidayDay: String, @field:Expose @field:SerializedName("HolidayMonth") val holidayMonth: String, @field:Expose @field:SerializedName("HolidayName") val holidayName: String,
                  @field:Expose @field:SerializedName("HolidayImageUrl") val holidayImageUrl: String, @field:Expose @field:SerializedName("ImageId") val imageId: Int, @field:Expose @field:SerializedName("HolidayBgColor") val holidayBgColor: String)